Invalid memory address or nil pointer dereference on validation

I ran `kube-aws validate --s3-uri s3://s3-bucket-name` and got this error:

Version 0.9.7-rc.3

```
# kube-aws validate --s3-uri s3://s3-bucket-name
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xdd0 pc=0x5f8f85]

goroutine 1 [running]:
panic(0xc66be0, 0xc42000e070)
    /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/kubernetes-incubator/kube-aws/core/root/config.ConfigFromBytes(0xc4201d8000, 0xde5d, 0xe05d, 0xde5d, 0xe05d, 0x0)
    /go/src/github.com/kubernetes-incubator/kube-aws/core/root/config/config.go:91 +0x265
github.com/kubernetes-incubator/kube-aws/core/root/config.ConfigFromFile(0xd927bf, 0xc, 0xd8d0b4, 0xc4201d6000, 0x0)
    /go/src/github.com/kubernetes-incubator/kube-aws/core/root/config/config.go:190 +0x77
github.com/kubernetes-incubator/kube-aws/core/root.ClusterFromFile(0xd927bf, 0xc, 0xd91a33, 0xb, 0xda81bd, 0x20, 0xda40cf, 0x1c, 0xda1a3a, 0x1a, ...)
    /go/src/github.com/kubernetes-incubator/kube-aws/core/root/cluster.go:110 +0x4d
github.com/kubernetes-incubator/kube-aws/cmd.runCmdValidate(0x13ddce0, 0xc4201ca3c0, 0x0, 0x2, 0x0, 0x0)
    /go/src/github.com/kubernetes-incubator/kube-aws/cmd/validate.go:46 +0x18f
github.com/kubernetes-incubator/kube-aws/vendor/github.com/spf13/cobra.(*Command).execute(0x13ddce0, 0xc4201ca360, 0x2, 0x2, 0x13ddce0, 0xc4201ca360)
    /go/src/github.com/kubernetes-incubator/kube-aws/vendor/github.com/spf13/cobra/command.go:620 +0x247
github.com/kubernetes-incubator/kube-aws/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x13dd3e0, 0x0, 0x0, 0x0)
    /go/src/github.com/kubernetes-incubator/kube-aws/vendor/github.com/spf13/cobra/command.go:699 +0x367
github.com/kubernetes-incubator/kube-aws/vendor/github.com/spf13/cobra.(*Command).Execute(0x13dd3e0, 0xc420169f50, 0xc420000340)
    /go/src/github.com/kubernetes-incubator/kube-aws/vendor/github.com/spf13/cobra/command.go:658 +0x2b
main.main()
    /go/src/github.com/kubernetes-incubator/kube-aws/main.go:10 +0x2d
```
I've seen similar errors referenced before, but they are marked as resolved. #202 #204

--- a/core/root/config/config.go
+++ b/core/root/config/config.go
@@ -88,6 +88,9 @@ func ConfigFromBytes(data []byte) (*Config, error) {
 	}
 
 	for i, np := range nodePools {
+		if np == nil {
+			return nil, fmt.Errorf("Empty nodepool definition found at index %d", i)
+		}
 		if err := np.Experimental.Taints.Valid(); err != nil {
 			return nil, fmt.Errorf("invalid taints for node pool at index %d: %v", i, err)
 		}

